Examples/Tests/silver_mueller/analysis_silver_mueller.py

```diff
@@ -24,15 +25,27 @@
 
 ds = yt.load( filename )
 all_data_level_0 = ds.covering_grid(level=0,left_edge=ds.domain_left_edge, dims=ds.domain_dimensions)
-Ex = all_data_level_0['boxlib', 'Ex'].v.squeeze()
-Ey = all_data_level_0['boxlib', 'Ey'].v.squeeze()
-Ez = all_data_level_0['boxlib', 'Ez'].v.squeeze()
+warpx_used_inputs = open('./warpx_used_inputs', 'r').read()
+if re.search('geometry.dims\s*=\s*RZ', warpx_used_inputs):
```

Comment:
Just a suggestion, maybe we could have an auxiliary boolean and use `re.search` only once:
```suggestion
geom_RZ = re.search('geometry.dims = RZ', warpx_used_inputs)
if geom_RZ:
```
Note that I also removed the `\s*` expression, the output in that file should always be written with one single white space, so this should work.

Reply:
ok that sounds good to me.  Making the change now